### ISSUE TITLE
Saturday is not shown in MS Edge!

### ISSUE THREAD

**Issue by Junbong** [Adphorus/react-date-range#255]
## Subject of the issue
All of Saturdays are not displayed in Microsoft Edge browser. Check out a screenshot below.

![2018-11-08 2 19 57](https://user-images.githubusercontent.com/6138662/48179415-68d6ae00-e362-11e8-9c48-97f9ccfa3289.png)

I tested in [Browserstack](browserstack.com):
* **Windows 10 / Edge 17 (latest)**
* **Windows 10 / Edge 16**
* **Windows 10 / Edge 15**
 
## [BUG] Bug Reproduce Steps
```jsx
// minDate is moment object - moment().subtract(1, 'month').startOf('day')
// maxDate is moment object - moment().endOf('day')
// startDate, endDate are also moment object

<DateRangePicker
  locale={DateRange.localeToAvailableLocale(locale)}
  ranges={[{ startDate, endDate, key: 'dateRanges' }]}
  maxDate={maxDate.toDate()}
  minDate={minDate.toDate()}
  onChange={this.handleChangeDate}
```

## [BUG] Expected behaviour
Saturday must displayed properly like other browsers.

Chrome:
![2018-11-08 2 38 08](https://user-images.githubusercontent.com/6138662/48179740-efd85600-e363-11e8-837a-ece5e1fcd075.png)

## Environment
Package Version: 1.0.0-beta
React version: 15.4.1
Node version: 8
Browser: MS Edge

**Comment by Junbong** [comment]
I tried to edit my code to change type both `startDate` and `endDate` to native `Date` object.

```jsx
<DateRangePicker
  locale={/* react-date-range/src/locale/enUS.js */}
  ranges={[{
    startDate: startDate.toDate(),
    endDate: endDate.toDate(),
    key: 'dateRanges',
  }]}
  maxDate={maxDate.toDate()}
  minDate={minDate.toDate()}
  onChange={this.handleChangeDate}
/>
```

My moment version is `2.11.2`.

**Comment by Junbong** [comment]
Can anybody answer it? It seems pretty critical issue..

**Comment by jeffyyyy** [comment]
@Junbong check out my pr https://github.com/Adphorus/react-date-range/pull/287#issue-259328752, fixed for edge

**Comment by karthikbalajikb** [comment]
Facing the same issue . @jeffyyyy i tried with your changes , doesn't worked .

**Comment by Junbong** [comment]
@jeffyyyy I have same problem on `1.0.0-beta-2`. See https://github.com/Adphorus/react-date-range/pull/287#issuecomment-501554562

**Comment by onurkerimov** [comment]
This issue is resolved in the latest release (1.1.0). Thanks to @jeffyyyy for fixing this! I'm closing this.
